core/src/main/scala/org/apache/spark/scheduler/ExecutorDecommissionInfo.scala

```diff
@@ -31,6 +31,8 @@ case class ExecutorDecommissionInfo(message: String, workerHost: Option[String]
  * State related to decommissioning that is kept by the TaskSchedulerImpl. This state is derived
  * from the info message above but it is kept distinct to allow the state to evolve independently
  * from the message.
+ *
+ * @since 3.1.0
```

Comment:
@holdenk @Ngone51 do we mean to expose this class to end-users?

Reply:
I don't think so. It's only used internally.

Reply:
Copy, will add @Private tag for it.